src/widgets/AppWindow.js

```diff
@@ -2656,58 +2699,58 @@ export default class AppWindow extends createjs.Container {
     this._contentArea.accessible.addChild(this.textFormat);
     let prefixText = 'Example of ';
     const PADDING = 7;
-    const boldText = new FormatText(prefixText, 'Bold Text', AccessibilityModule.ROLES.FORMAT_TEXT_BOLD, 'bold', '22px', 'Arial');
+    const boldText = new FormatText(prefixText, 'Bold Text', AccessibilityModule.ROLES.FORMAT_TEXT_BOLD, 'bold', '22', 'Arial');
```

Comment:
Is it correct to remove `px` from all these strings?  It looks like FormatText line 25 (`const label = new createjs.Text(value, `${fontType} ${fontSize} ${fontFamily}`)`) needs it.

Reply:
Added string `px`